arangod/Aql/IResearchViewNode.cpp

```diff
@@ -681,28 +681,34 @@ ViewSnapshotPtr snapshotSingleServer(IResearchViewNode const& node,
     if (options.forceSync) {
       syncViewSnapshot(*snapshot, view.name());
     }
-    if (options.restrictSources) {
-      return std::make_shared<ViewSnapshotView>(*snapshot, options.sources);
-    }
+    // if (options.restrictSources) {
+    //  return std::make_shared<ViewSnapshotView>(*snapshot, options.sources);
+    //}
```

Comment:
Still needed?

Reply:
We want to restore Snapshot "views" in future PRs. Just now it is broken :( , so we disabled this code temporary